Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM alpine:latest
+FROM python:3.8-alpine
 
-RUN apk update && apk add python3 && mkdir -p /opt/amazon-mws-analytics
-COPY . /opt/amazon-mws-analytics
+WORKDIR /app
 
-WORKDIR /opt
-RUN cd /opt/amazon-mws-analytics && pip3 install -r requirements.txt
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
 
-ENTRYPOINT python3 -m amazon-mws-analytics
+COPY *.py ./
+
+WORKDIR /
+CMD ["python", "-m", "app"]
```

Comment:
Your Docker-fu is getting strong @sctice!

I'm little off-put by the `WORKDIR /` though. The `-m` flag implicitly running `__main__.py` surprises me. Not that it's wrong, its just a little surprising.

Reply:
I hear you. I just copied the previous invocation because it works and using relative imports was breaking in unexpected ways when I tried to run the main script directly. I suspect that might not be an issue after creating `__init__.py`, but I'm not positive.